eng/pipelines/libraries/stress/http.yml

```diff
@@ -0,0 +1,73 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+    - "*"
+
+schedules:
+- cron: "0 13 * * *" # 1PM UTC => 5 AM PST
+  displayName: HttpStress nightly run
+  branches:
+    include:
+    - master
+
+variables:
+  - template: ../variables.yml
+  - name: dockerfilesFolder
+    value: $(Build.SourcesDirectory)/eng/docker
+  - name: httpStressProject
+    value: $(sourcesRoot)/System.Net.Http/tests/StressTests/HttpStress
+  - name: sdkBaseImage
+    value: dotnet-sdk-libraries-current
+
+
+jobs:
+
+- job: linux
+  displayName: Docker Linux
+  pool:
+    name: Hosted Ubuntu 1604
+
+  steps:
+  - checkout: self
+    clean: true
+    fetchDepth: 1
```

Comment:
Why set this? I know it is supposed to be an optimization of some kind. But I've seen other discussions where using 'fetchDepth' is causing builds to fail because the number is large enough. What happens when a PR is many commits in depth?

My suggestion is to not use this unless you see that the pipeline is running super slow.

Reply:
I agree, we should increase this value to 5 and later (or now) share the value in the global variables template.

Reply:
I can change to 5. I've generally found that shallow checkouts greatly improve build times for sufficiently large repos.

Reply:
We use 5 elsewhere and I think consistency makes sense here (as we are checking out the same repo).